routes/web.php

```diff
@@ -24,23 +24,31 @@
     // Route::get('/alletermine', 'MatchdayController@index')->name('home');
 
     Route::get('/termine/erstellen', 'MatchdayController@create')->name('matchdays.create');
+    Route::post('/termine/erstellen', 'MatchdayController@store')->name('matchdays.store');
 
-    Route::get('/termine/bearbeiten', 'MatchdayController@edit')->name('matchdays.edit');
+    Route::get('/termine/bearbeiten/{matchday}', 'MatchdayController@edit')->name('matchdays.edit');
+    // // Route::get('/termine/{matchday}/bearbeiten', 'MatchdayController@edit')->name('matchdays.edit');
+    Route::patch('/termine/{matchday}', 'MatchdayController@update')->name('matchdays.update');
+    Route::delete('/termine/{matchday}', 'MatchdayController@destroy')->name('matchdays.destroy');
 
 
     Route::get('/allespieler', 'PlayerController@index')->name('players.index');
 
     Route::get('/spieler/erstellen', 'PlayerController@create')->name('players.create');
-
-    Route::get('/spieler/bearbeiten', 'PlayerController@edit')->name('players.edit');
+    Route::post('/spieler/erstellen', 'PlayerController@store')->name('players.store');
 
     Route::get('/spieler/{player}', 'PlayerController@show')->name('players.show');
+    Route::get('/spieler/bearbeiten/{player}', 'PlayerController@edit')->name('players.edit');
+    // Route::get('/spieler/{player}/bearbeiten', 'PlayerController@edit')->name('players.edit');
```

Comment:
Are all these routes necessary?CRUD stuff can often be handled by Route::resource